runtime/doc/news.txt

```diff
@@ -117,15 +117,6 @@ The following changes may require adaptations in user config or plugins.
   list is documented in |treesitter-highlight-groups|.
 
 
-==============================================================================
-BREAKING CHANGES IN HEAD                                    *news-breaking-dev*
-
-The following breaking changes were made during the development cycle to
-unreleased features on Nvim HEAD.
-
-• ...
-• ...
-
```

Comment:
Any reason/discussion/context for removing this? just because not used (*yet*)?

Reply:
To be clear, the intention has always been to remove this section before release, so this can just be a dumping ground for "hey, pay attention!"

But since pretty much nobody pays attention anyway...

Reply:
> `news.txt` is supposed to be only for changes between releases.

Not sure where that idea came from. The reason we were able to close the "following head" issue https://github.com/neovim/neovim/issues/14090 is because we intended to use news.txt to track HEAD changes.

There is no risk of "clusterfuck". It's just one section that can be updated, and deleted at release-time.

Reply:
Alright, I'll revert it then.

Reply:
> Somehow this got overlooked during the original news.txt effort.

Just for the record: Not quite; the "original news.txt effort" aimed at something different (replacing the release newsletter, or at least making it easier to generate). "We don't need Following HEAD now" was a later, separate, decision.

Reply:
Yes, that was the impression I got as well which is why I had the idea that news.txt wasn't for HEAD changes. In any case, I don't mind testing out this format and see if it works for us.